RadarTarget[] targets
RadarTarget[] gt_targets
